src/napari_spatialdata/utils/_test_utils.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PyQt6.QtCore import Qt
```

Comment:
We shouldn't directly import from PyQt5/6 or pyside. Users should be able to use either Qt bindings. We should instead use qtpy as the compatibilty layer. Please update all Qt-related imports to use `qtpy`

See here for an example: https://github.com/napari/napari/blob/e1415c59ca3a1de78ac7e31784edd46a04155b9e/napari/_qt/layer_controls/qt_image_controls.py#L3-L11

Reply:
In case you're curious the readme does a good job explaining why we use `qtpy`: https://github.com/spyder-ide/qtpy

Reply:
Also, do we support Qt6? It isn't even really fully supported in napari yet (cc @giovp )

Reply:
Is this why you needed to patch the docs for Qt6 in #107

Reply:
No, the docs `conf.py` stated already that PyQt6 is being used. The doc builds in other PRs were also failing for the same reason as I suspect that PyQt6 dependencies changed. The updated dependency was not supported on Ubuntu 20.04.

Reply:
Resolved with [b6953f4](https://github.com/scverse/napari-spatialdata/pull/99/commits/b6953f44a00101034b47a2a01d4121046c988451)